spring-integration-core/src/main/java/org/springframework/integration/channel/NullChannel.java

```diff
@@ -27,32 +32,144 @@
  * All receive() calls will return <em>null</em>, and all send() calls
  * will return <em>true</em> although no action is performed.
  * Note however that the invocations are logged at debug-level.
- * 
+ *
  * @author Mark Fisher
+ * @author Gary Russell
  */
-public class NullChannel implements PollableChannel {
+@ManagedResource
+public class NullChannel implements PollableChannel, MessageChannelMetrics, NamedComponent {
```

Comment:
Isn't related to this PR, but why `NullChannel` is `PollableChannel`? Why just `MessageChannel` isn't enough ?